src/extension.ts

```diff
@@ -354,6 +363,47 @@ export function activate(context: vscode.ExtensionContext) {
     }));
 
     //
+    // Import of a GLB file and writing out its various chunks.
+    //
+    context.subscriptions.push(vscode.commands.registerCommand('gltf.importGlbOpen', () => {
+        const options: vscode.OpenDialogOptions = {
+             canSelectMany: false,
+             openLabel: 'Import',
+             filters: {
+                'Binary glTF': ['glb'],
+                'All files': ['*']
+            }
+        };
+
+        vscode.window.showOpenDialog(options).then(fileUri => {
+            if (fileUri && fileUri[0]) {
+                try {
+                    GlbImport.load(fileUri[0].fsPath);
+                } catch (ex) {
+                    vscode.window.showErrorMessage(ex.toString());
+                }
+            }
+        });
+    }));
+
+    context.subscriptions.push(vscode.commands.registerCommand('gltf.importGlbFile', (fileUri) => {
+
+        if (typeof fileUri == 'undefined' || !(fileUri instanceof vscode.Uri)) {
+            if (vscode.window.activeTextEditor === undefined) {
+                vscode.commands.executeCommand('gltf.importGlbOpen');
+                return;
+            }
+            fileUri = vscode.window.activeTextEditor.document.uri;
+        }
+
+        try {
+            GlbImport.load(fileUri.fsPath);
```

Comment:
Here, we're calling `GlbImport.load`, which is async, so we must await if we want to catch errors that have happened after awaiting things in this function.

```
            await GlbImport.load(fileUri.fsPath);
```